test/advection_stencil_test.jl

```diff
@@ -124,11 +169,12 @@ end
                             for (d3, Δz) in Δz_opts
                                 @testset "Δz $d3" begin
                                     dudt = [stencil(
-                                                u0[(i - lpad):(i + rpad)],
+                                                u0[(i - lpad):(i + rpad)].*Δz[(i - lpad):(i + rpad)],
                                                 v[i:(i + 1)], Δt,
-                                                Δz[i])
+                                                Δz[(i - lpad):(i + rpad)])
```

Comment:
In general we want all the stencils to have the same function signature, so they can be used interchangeably, so if the ppm solver needs an array of delta z values to work, we should change all the other stencils to also take an array of delta z values, even if they don't need to use that extra information.